### PR TITLE
fix: fix middleware service clone.

### PR DESCRIPTION
**Pull request by andysim3d**
[Closes/Fixes] #

reference: https://docs.rs/tower/latest/tower/trait.Service.html#be-careful-when-cloning-inner-services

## Proposed Changes

  - 
  -
  -

### CODE REVIEW

**Review by 0xfourzerofour — APPROVED** [review]

File: bin/rundler/chain_specs/opbnb_testnet.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name = "Opbnb Testnet"
+id = 5611
+
+priority_fee_oracle_type = "USAGE_BASED"
+min_max_priority_fee_per_gas = 30000000000
+bloxroute_enabled = true
```
Comment (dancoombs) [inline bin/rundler/chain_specs/opbnb_testnet.toml:6]: Why is this true?

File: bin/rundler/chain_specs/opbnb_testnet.toml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+name = "Opbnb Testnet"
+id = 5611
+
+priority_fee_oracle_type = "USAGE_BASED"
+min_max_priority_fee_per_gas = 30000000000
```
Comment (dancoombs) [inline bin/rundler/chain_specs/opbnb_testnet.toml:5]: Just wondering, where did this come from?

Comment (andysim3d) [reply bin/rundler/chain_specs/opbnb_testnet.toml:5]: this is for local test only. I copied it from  polygon.

Comment (dancoombs) [reply bin/rundler/chain_specs/opbnb_testnet.toml:5]: Got it, if just used for your local testing we should remove this from the repo.

**Pull request merged**